dashboards/MySQL/MySQL_Instance_Summary.json

```diff
@@ -6206,15 +6281,12 @@
         "notice": false,
         "now": true,
         "refresh_intervals": [
+            "1s",
```

Comment:
Let's double check the `1s` interval. @puneet0191 What do you think?

Reply:
@anton-bystrov we have default 1m as refresh interval on most of the dashboards, why use 1s here?

Reply:
Here I can't say why interval 1s. I'll change it to 1m.

Reply:
I removed interval 1 second at all dashboards.

Reply:
I'd definitely leave it to the discretion of QA team. We need to test and confirm that this change is fine.